code/modules/blob/blob.dm

```diff
@@ -4,16 +4,21 @@
 	icon = 'icons/mob/blob.dmi'
 	icon_state = "blob"
 	light_range = 3
+	light_color = "#b5ff5b"
 	desc = "Some blob creature thingy"
 	density = 1
 	opacity = 0
 	anchored = 1
 	mouse_opacity = 2
+	layer = 4
 
 	var/maxHealth = 30
 	var/health
+	var/regen_rate = 5
 	var/brute_resist = 4
+	var/laser_resist = 5	// Special resist for laser based weapons - Emitters or handheld energy weaponry. Damage is divided by this and THEN by fire_resist.
```

Comment:
Did a bit of quick testing to confirm, i'm right about flamethrowers. Baymerge has not fixed them, they are still broken and worthless.

The fire from a flamethrower does nothing to a blob. Literally nothing, zero damage. The only thing that does damage is bludgeoning a blob with the thing, and thats still barely enough to kill a piece of a blob before it eats you